src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categoryList: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    setStatus: (state, { payload }) => {
+      /* eslint-disable no-param-reassign */
+      state.categoryList = payload === 'Under construction' ? 'Under construction' : state.categoryList;
```

Comment:
- Good job done so far, but it is not advisable to disable linters, remember that linters help us ensure that we write code that is easy to maintain and meets global standards, please fix this 🙏